uiowa/uiowa
# Event split: Attendance Required Field

Resolves #6469


# How to test

```
ddev blt ds --site=iowasummerwritingfestival.uiowa.edu && ddev blt ds --site=oniowa.uiowa.edu && ddev blt ds --site=admissions.uiowa.edu && ddev drush @iowasummerwritingfestival.local uli node/add/event && ddev drush @oniowa.local uli node/add/event && ddev drush @admissions.local uli node/add/event
```
- Review code and verify that enough defensive checks are in place so that an update hook is unnecessary. 
- Verify that the new "Attendance Required" field is available on all of the above sites. 
- Verify that checking the box will render an "Attendance Required" badge on the full event node and the /events teaser list for oniowa and iowasummerwriting festival.
- Verify that the event status badge has moved to the bottom of the card to display inline with the  "Attendance Required" badge 
- Test all fields and their teaser and full page display on oniowa, admissions, and iowasummerwriting festival to check for regressions like https://github.com/uiowa/uiowa/pull/6495.
